_data/whats-new.yml

```diff
@@ -4,8 +4,38 @@ description: |
   We exclude from this list proofreading, spelling checks, and all minor updates.
 link: /whats-new.html
 thread: /whatsnew-feed.xml
-updated: Mon May 13 11:36:22 2019
+updated: Mon May 20 11:21:53 2019
 entries:
+- description: Added [reference documentation](https://devdocs.magento.com/guides/v2.3/extension-dev-guide/routing.html#result-object)
+    about the `result object` returned by the `Action` class.
+  versions: 2.1.x, 2.2.x, 2.3.x
+  type: Major update
+  date: May 20, 2019
+  link: https://github.com/magento/devdocs/pull/4541
+- description: Added examples of [creating and updating a Magento admin user](https://devdocs.magento.com/guides/v2.3/install-gde/install/cli/install-cli-subcommands-admin.html)
+    with the command line.
+  versions: 2.1.x, 2.2.x, 2.3.x
+  type: Major update
+  date: May 17, 2019
+  link: https://github.com/magento/devdocs/pull/4526
+- description: Added reference documentation for [ZIP code attributes](https://devdocs.magento.com/guides/v2.3/howdoi/checkout/checkout_zip.html),
```

Comment:
The `, the same country` part doesn't make sense or maybe it can be worded better?

Reply:
one line was missing